Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TetrahedronFEMForceField.h

```diff
@@ -225,12 +225,20 @@ class TetrahedronFEMForceField : public core::behavior::ForceField<DataTypes>, p
     Data<type::vector<Real> > _vonMisesPerElement; ///< von Mises Stress per element
     Data<type::vector<Real> > _vonMisesPerNode; ///< von Mises Stress per node
     Data<type::vector<type::Vec4f> > _vonMisesStressColors; ///< Vector of colors describing the VonMises stress
-    
+
+    Real _minVMN;
+    Real _maxVMN;
+
+    Data<bool> _showForceField; ///< draw the force field for the current object
```

Comment:
I think this is related to a more fundamental issue that the proper design for such kind of rendering feature is to implement the rendering code in separated component inheriting from visualmodel instead of implementing that in the draw() method. Include the showXXX data. The draback is that Sofa is not doing such a thing consistently.